Support GraphQLTypeReference implementing GraphQLInputType.

brimworks opened the pull request:
This resolves:

https://github.com/graphql-java/graphql-java/issues/247

...and would allow us to define recursive input types.

brimworks commented:
Looks like this is already done here: https://github.com/graphql-java/graphql-java/pull/248

brimworks commented:
...or I meant to say here: https://github.com/graphql-java/graphql-java/pull/204

dminkovsky commented:
Take a look and see what you think of that one. I saw a commit was pushed
to that yesterday but I don't know what it included. I was hoping tests :)

ср, 23 нояб. 2016 г. в 18:36, Brian Maher <notifications@github.com>:

> Looks like this is already done here: #248
> <https://github.com/graphql-java/graphql-java/pull/248>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/graphql-java/graphql-java/pull/248#issuecomment-262655941>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANWZbq00lWXlKuHkGPO2b-5_jsKjJphks5rBM4AgaJpZM4K7JWA>
> .
>

brimworks commented:
Thanks for the heads up @dminkovsky, I have commented on this pull request:

https://github.com/graphql-java/graphql-java/pull/204

...basically it looks like two changes are getting bundled into one PR and still no tests and no schema validation (is schema validation important to you?).

If that PR isn't moving along by Monday, then I'll open a separate PR that addresses the issues (and bundles only a single change).

Thanks,
-Brian

kaqqao commented:
It was me who accidentally pushed unrelated stuff. Reverted the wrong commits in the meantime, and added basic tests and validation.
